docs/01-getting-started/01-installation.mdx

```diff
@@ -39,7 +39,7 @@ Next.js now ships with TypeScript, ESLint, and Tailwind CSS configuration by def
 
 After the prompts, `create-next-app` will create a folder with your project name and install the required dependencies.
 
-> **Good to know**: While you can use the [Pages Router](/docs/pages) in your new project. We recommend starting new applications with the App Router to leverage React's latest features.
+> **Good to know**: While you can use the [Pages Router](/docs/pages) in your new project, we recommend starting new applications with the App Router to leverage React's latest features.
 
```

Comment:
```suggestion
```

I'd say it's best to remove this entirely. Let's let the developer make this choice.